lib/displayHelpers.ts

```diff
@@ -37,32 +37,30 @@ const thinSpace = "\u202F";
 const printableCoin = (coin: Coin, chainInfo: ChainInfo) => {
   if (!coin.amount || !coin.denom) return "";
 
-  // The display denom from configuration
-  if (coin.denom === chainInfo.denom) {
-    const exponent = Number(chainInfo.displayDenomExponent);
-    const value = Decimal.fromAtomics(coin.amount ?? "0", exponent).toString();
-    const ticker = chainInfo.displayDenom;
-    return value + thinSpace + ticker;
-  }
-
-  // Auto-convert leading "u"s
-  if (coin.denom.startsWith("u")) {
-    const value = Decimal.fromAtomics(coin.amount ?? "0", 6).toString();
-    const ticker = coin.denom.slice(1).toUpperCase();
-    return value + thinSpace + ticker;
-  }
-
   // Ellide IBC tokens
   if (coin.denom.startsWith("ibc/")) {
     const value = coin.amount;
     const hash = coin.denom.slice(4);
     const ellidedHash = ellideMiddle(hash, 11);
-    const ticker = `ibc/${ellidedHash}`;
+    const ticker = `ibc/${ellidedHash}`.toUpperCase();
+    return value + thinSpace + ticker;
```

Comment:
Here I'd use the native notation with lower case ibc and upper case hash, e.g. `ibc/D189335C6E4A68B513C10AB227BF1C1D38C746766278BA3EEB4FB14124F1D858`